storm-server/src/main/java/org/apache/storm/daemon/nimbus/Nimbus.java

```diff
@@ -2162,7 +2165,12 @@ private boolean isReadyForMKAssignments() throws Exception {
     }
 
     private void mkAssignments() throws Exception {
-        mkAssignments(null);
+        try {
+            mkAssignments(null);
```

Comment:
Do we want to make the change inside `mkAssignments(String scratchTopoId)` so we can count errors during the rebalance command as well?

Reply:
I can change that.